torchx/Makefile

```diff
@@ -5,11 +5,29 @@ TORCHX_SO = $(PRIV_DIR)/torchx.so
 # CMAKE_BUILD_FLAGS = --verbose
 CMAKE_BUILD_DIR ?= $(MIX_APP_PATH)/cmake
 C_SRC = $(shell pwd)/c_src
+NERVES_BUILD ?= nerves_build_no
+
+# this .cmake file makes this project compatible with nerves and it should have no effect on normal build
+TOOLCHAIN_FILE ?= $(shell pwd)/nerves/toolchain.cmake
 
 .DEFAULT_GLOBAL := build
 
 build: check_libtorch $(TORCHX_SO)
 
+nerves_build_yes:
+	@ mkdir -p $(PRIV_DIR)
+	@ if [ "$(PRIV_DIR)/libtorch" != "$(abspath $(PRIV_DIR)/libtorch)" ]; then \
+		rm -rf "$(PRIV_DIR)/libtorch" ; \
+		cp -a "$(abspath $(LIBTORCH_DIR)/lib)" "$(PRIV_DIR)/libtorch" ; \
+	fi
+
+nerves_build_no:
```

Comment:
Instead of `nerves_build_yes/no`, we could go with a more general `generate_symlinks_yes/no` terminology.
Also, instead of having 2 make rules, perhaps we could improve the internal `if` to take the `NERVES_BUILD` variable (which could be called `USE_SYMLINKS`) into account